architect.yml

```diff
@@ -15,16 +15,18 @@ services:
   app:
     build:
       context: ./
-      ${{ if architect.environment == 'local' }}:
-        args:
-          DEBUG: 'true'
```

Comment:
If you're going to remove the DEBUG arg then you need to update the Dockerfile to use NODE_ENV:

https://github.com/architect-templates/nuxt/blob/main/Dockerfile#L10